torch/utils/cpp_extension.py

```diff
@@ -1056,6 +1058,20 @@ def _jit_compile(name,
                  verbose,
                  with_cuda,
                  is_python_module):
+    if with_cuda is None:
+        with_cuda = any(map(_is_cuda_file, sources))
+    with_cudnn = any(['cudnn' in f for f in extra_ldflags or []])
+    if IS_HIP_EXTENSION and with_cuda or with_cudnn:
```

Comment:
Did you mean `IS_HIP_EXTENSION and (with_cuda or with_cudnn)`? Because that's not what you've written here ;)

Reply:
Right. :) Thanks!